components/kyma-metrics-collector/pkg/process/event_stream.go

```diff
@@ -116,6 +118,16 @@ func (inp Input) Parse(providers *Providers) (*edp.ConsumptionMetrics, error) {
 			if infraConfig.Networks.VNet.CIDR != nil {
 				vnets += 1
 			}
+		case AWS:
+			decoder := serializer.NewCodecFactory(scheme.Scheme).UniversalDecoder()
+			infraConfig := &gardenerawsv1alpha1.InfrastructureConfig{}
+			err := runtime.DecodeInto(decoder, rawExtension.Raw, infraConfig)
+			if err != nil {
+				return nil, err
+			}
+			if infraConfig.Networks.VPC.CIDR != nil {
+				vnets += 1
+			}
```

Comment:
Why in case of **vpc** we increment **vnets**?

Reply:
As the [schema](https://github.com/kyma-project/control-plane/tree/main/components/kyma-metrics-collector#data-collection) expects `vnets` which inturn is used by other systems. Hence we had to use `vnets` for AWS as well